relay/relay.go

```diff
@@ -49,7 +50,7 @@ type ClientOptions struct {
 	RelayDebugUrl   string
 }
 
-func NewClient(localURL *url.URL, opts *ClientOptions) *Client {
+func NewClient(token string, localURL *url.URL, opts *ClientOptions) *Client {
```

Comment:
Isn't token optional? Or is it just empty when not set?

Reply:
Token is required, we generate one in the listen cli command if we fail to grab one from the config file